numpyro/contrib/nn.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from flax import nn as flax_nn
```

Comment:
@TuanNguyen27 I guess I made you confused. Here I just meant to move those imports into `flax_module` function, kind of
```python
def flax_module(...):
    try:
        import flax
    except:
        ...
```

With this, all `flax_module`, `haiku_module`, `random_module` primitives are gathered into a single namespace `numpyro.contrib.nn`. By the way, is it better to rename this namespace to `numpyro.contrib.module` (because `numpyro.nn` is used for nn layers, rather than primitives)?

Reply:
ahhh i see what you mean now! Does this module still need `__init__.py` ? We can leave that empty too right ?

Reply:
> Does this module still need __init__.py?

If you make `module` a folder then `__init__` is necessary but you need to expose `flax_module`, `haiku_module`, `random_module` functions to this init file (so users can do `from numpyro.contrib.module import ...` - without doing so, users have to do `from numpyro.contrib.module.flax_module import flax_module`). Because the implementation is short, I prefer to put `flax_module`, `haiku_module`, `random_module` functions in `contrib/module.py` file for simplicity. But it depends on you.